app/assets/stylesheets/answers.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Answers controller here.
```

Comment:
You can go either way with it. It is probably a good idea to learn to do it both ways since you will find a lot of code that still uses scss files rather than css in the javascript.